backend/app/services/adapters/public_retriever.py

```
@@ -0,0 +1,296 @@
+                )
+
+        # Update Kind record
+        kind.name = new_name
+        kind.namespace = new_namespace
+        kind.json = retriever.model_dump(mode="json", exclude_none=True)
+        db.commit()
+        db.refresh(kind)
+        return RetrieverAdapter.to_retriever_dict(kind)
+
+    def delete_retriever(
+        self, db: Session, *, retriever_id: int, current_user: User
```

Ruff (0.14.10)

Unused method argument: `current_user`

(ARG002)